### DISCUSSION TITLE
PII Redaction - OK in Contact Lens but not redacted in SCV Conversation

### DISCUSSION THREAD

**Discussion by dschueler** [amazon-connect/amazon-connect-salesforce-scv#167] [Q&A]
I have turned on PII redaction in the Contact Flow and that all works fine as far as Contact Lens is concerned. However if I look at the Conversation in Salesforce it seems to pull an unredacted version. I THINK this is being pulled from AWS and not stored in SF. If thats the case it seems like its pulling the unredacted copy. Is there a way to make what shows in the Salesforce Conversation display be the redacted transcription?

**Comment by dougjaso** [comment]
SF is writing the real-time conversation transcription to the conversation record in Salesforce. PII redaction happens post call. This is why you are seeing the redacted version in Connect UI, but the unredacted version in SF. SF is working on a fix for this. You would need to engage Salesforce to determine the status of that fix.

Sent from my iPhone

On Apr 5, 2024, at 7:11 AM, dschueler ***@***.***> wrote:

﻿

I have turned on PII redaction in the Contact Flow and that all works fine as far as Contact Lens is concerned. However if I look at the Conversation in Salesforce it seems to pull an unredacted version. I THINK this is being pulled from AWS and not stored in SF. If thats the case it seems like its pulling the unredacted copy. Is there a way to make what shows in the Salesforce Conversation display be the redacted transcription?
